Pynq-Z1/bitstream/base.tcl

```diff
@@ -176,7 +176,7 @@ proc create_hier_cell_mb3_timers_subsystem { parentCell nameHier } {
   variable script_folder
 
   if { $parentCell eq "" || $nameHier eq "" } {
-     catch {common::send_msg_id "BD_TCL-102" "ERROR" "create_hier_cell_mb3_timers_subsystem() - Empty argument(s)!"}
+     catch {common::send_msg_id "BD_TCL-102" "ERROR" create_hier_cell_mb3_timers_subsystem() - Empty argument(s)!"}
```

Comment:
we discussed offline - this is what Vivado is generating and we'll monitor if issue persists when we upgrade Vivado versions later.